tests/urls.py

```diff
@@ -11,6 +12,10 @@
     path("django-admin/", admin.site.urls),
     path("admin/", include(wagtailadmin_urls)),
     path("documents/", include(wagtaildocs_urls)),
+]
+
+urlpatterns += i18n_patterns(
```

Comment:
it would be good to add a note, or even section in the README about usage with translation